oximeter/collector/src/lib.rs

```diff
@@ -4,35 +4,71 @@
 
 //! Implementation of the `oximeter` metric collection server.
 
-// Copyright 2021 Oxide Computer Company
-
-use dropshot::{
-    endpoint, ApiDescription, ConfigDropshot, ConfigLogging, HttpError,
-    HttpResponseUpdatedNoContent, HttpServer, HttpServerStarter,
-    RequestContext, TypedBody,
-};
-use internal_dns::resolver::{ResolveError, Resolver};
+// Copyright 2023 Oxide Computer Company
+
+use anyhow::anyhow;
```

Comment:
Same question here as elsewhere -- looks like a bunch of collapsed imports got flattened.